_sources/Wavefont.designspace

```diff
@@ -3,13 +3,17 @@
     <axes>
         <axis default="100" minimum="0" maximum="100" name="Roundness" tag="ROND"/>
         <axis default="-100" minimum="-100" maximum="100" name="Vertical Element Alignment" tag="YELA">
-            <map input="-100" output="0"/>
-            <map input="100" output="1"/>
```

Comment:
@RosaWagner this maps -100..+100 user-space YELA values to 0..1 design-space alignment values, I don't think it can be removed, is there a specific reason for that?

Reply:
Ah I have confuse output and input. I saw weird fractional coordinates in the FVAR but that was maybe another axis.